python-sdk/src/astro/databases/base.py

```diff
@@ -416,6 +429,8 @@ def load_file_to_table(
         columns_names_capitalization: ColumnCapitalization = "original",
         enable_native_fallback: bool | None = LOAD_FILE_ENABLE_NATIVE_FALLBACK,
         load_options: LoadOptions | None = None,
+        databricks_job_name: str = "",  # skipcq PYL-W0613
+        pandas_options: PandasLoadOptions | None = None,
```

Comment:
@pankajastro What do you think of us using the `load_options` argument instead of introducing a pandas-specific one?

Are there any scenarios where we want both of them?

 @dimberman , since you recently worked on this, I'd love your thoughts as well

Reply:
Yes, I was hoping that when I suggested @dimberman to add `load_options`, it would become more generic so we can add `PandasLoadOption` and pass it to `load_options` itself and not a separate param

Reply:
+1, would love if we could do that in this PR.

Reply:
Let's say we are doing native load and want to create schema and table with auto-detection (method `create_schema_and_table_if_needed`) then we might need pandas options for this and the load_option for other native supported param?? @utkarsharma2 @tatiana WDYT please let me know after that I'm ok to remove this and use load_options

Reply:
@pankajastro Can we address this in a separate PR? Since we haven't reached a conclusion on the earlier discussion.

Reply:
sure, @utkarsharma2 thanks.

Reply:
Please, let's do it as part of this PR. We are moving from `native_kwargs` to `load_options`. To introduce a third way of doing this (`pandas_otions, even temporarily to our `main` branch, could be a step backwards). I don't believe we should merge this PR before renaming this argument.

Reply:
@utkarsharma2, if you could add a link to the follow-up ticket to change `load_options` to a list, it would be great!

Reply:
@tatiana, Here - https://github.com/astronomer/astro-sdk/issues/1532